commands/FBAccessibilityCommands.py

```diff
@@ -0,0 +1,108 @@
+    #Starting accessibility server is different for simulator and device
+    if fb.evaluateBooleanExpression('((NSRange)[(NSString *)[(id)[(id)[UIDevice currentDevice] model] lowercaseString] rangeOfString:@"simulator"]).location != __LONG_MAX__'):
+      lldb.debugger.HandleCommand('expr (void)[[UIApplication sharedApplication] accessibilityActivate]')
+    else:
+      lldb.debugger.HandleCommand('expr (void)[[[UIApplication sharedApplication] _accessibilityBundlePrincipalClass] _accessibilityStartServer]')
+
+def accessibilityLabel(view):
+  #using Apple private API to get real value of accessibility string for element.
+  return fb.evaluateExpressionValue('(id)[%s accessibilityAttributeValue:%i]' % (view, ACCESSIBILITY_LABEL_KEY), False)
+
+def printAccessibilityHierarchy(view, indent = 0):
+  a11yLabel = accessibilityLabel(view)
+  classDesc = objHelpers.className(view)
+  indentString = '   | ' * indent
+
+  #if we don't have any accessibility string - we should have some children
+  if int(a11yLabel.GetValue(), 16) == 0:
+    print indentString + ('{} {}'.format(classDesc, view))
+    #We call private method that gives back all visible accessibility children for view
+    accessibilityElements = fb.evaluateExpression('(id)[[[UIApplication sharedApplication] keyWindow] _accessibilityElementsInContainer:0 topLevel:%s includeKB:0]' % view)
+    accessibilityElementsCount = int(fb.evaluateExpression('(int)[%s count]' % accessibilityElements))
+    for index in range(0, accessibilityElementsCount):
+      subview = fb.evaluateExpression('(id)[%s objectAtIndex:%i]' % (accessibilityElements, index))
+      printAccessibilityHierarchy(subview, indent + 1)
+  else:
+    print indentString + ('({} {}) {}'.format(classDesc, view, a11yLabel.GetObjectDescription()))
+
+def accessibilityGrepHierarchy(view, needle):
+  foundLocally = False
+  a11yLabel = accessibilityLabel(view)
+
+  #if we don't have any accessibility string - we should have some children
+  if int(a11yLabel.GetValue(), 16) == 0:
+    #We call private method that gives back all visible accessibility children for view
+    accessibilityElements = fb.evaluateExpression('(id)[[[UIApplication sharedApplication] keyWindow] _accessibilityElementsInContainer:0 topLevel:%s includeKB:0]' % view)
+    accessibilityElementsCount = int(fb.evaluateExpression('(int)[%s count]' % accessibilityElements))
+    for index in range(0, accessibilityElementsCount):
+      subview = fb.evaluateExpression('(id)[%s objectAtIndex:%i]' % (accessibilityElements, index))
+      foundLocally |= accessibilityGrepHierarchy(subview, needle)
+  elif re.match(r'.*' + needle + '.*', a11yLabel.GetObjectDescription(), re.IGNORECASE):
+    classDesc = objHelpers.className(view)
+    print('({} {}) {}'.format(classDesc, view, a11yLabel.GetObjectDescription()))
+
+    #First element that is found is copied to clipboard
+    if not foundLocally:
```

Comment:
It seems this will always be true, since `foundLocally` will always be false in this `elif` branch.